components/nexusoperations/config.go

```diff
@@ -178,7 +178,7 @@ func ConfigProvider(dc *dynamicconfig.Collection) *Config {
 				keys[i] = strings.ToLower(k)
```

Comment:
maybe this is the problem? it shouldn't be modifying the input slice. and definitely shouldn't be modifying `defaultDisallowedOperationHeaders`